src/server/database/sentences.ts

```diff
@@ -48,17 +48,23 @@ export default class Sentences {
         const pool = await this.sql.createPool();
         return Promise.all(
             batch.sentences.map((sentence: string) => {
+                const items = sentence.split('\t');
+                // handle files with text, source, age
+                const text = items[0];
```

Comment:
What if _items[0]_ is undefined??

Reply:
Good catch! No I think the setup is not that robust. I'll add a try catch :)